Dockerfile

```diff
@@ -2,12 +2,18 @@
 # Use of this source code is governed by an Apache2
 # license that can be found in the LICENSE file.
 
-FROM scratch
+FROM alpine:latest
 
 MAINTAINER Torin Sandall <torinsandall@gmail.com>
 
-ADD /opa_linux_GOARCH /opa
+ENV BIN /usr/bin/opa
```

Comment:
[minor nitpick] Is /usr/bin the right place for this? I'm not super familiar with alpine.